TypeError: manualChunks is not a function の解消

https://github.com/sweetpotato/ijinden-deck-builder/actions/runs/25966969614/job/76332139081

```
Run npm run build
> ijinden-deck-builder@0.0.0 build
> vite build
vite v8.0.13 building client environment for production...
Warning: Invalid output options (1 issue found)
- For the "manualChunks". Invalid type: Expected Function but received Object. 

transforming...✓ 464 modules transformed.
<anonymous_script>:0
TypeError: manualChunks is not a function
    at name (file:///home/runner/work/ijinden-deck-builder/ijinden-deck-builder/node_modules/rolldown/dist/shared/rolldown-build-BVD3dIdE.mjs:3067:10)
    at name (file:///home/runner/work/ijinden-deck-builder/ijinden-deck-builder/node_modules/rolldown/dist/shared/rolldown-build-BVD3dIdE.mjs:3078:54)
Node.js v24.15.0
Error: Process completed with exit code 1.
```

--- a/vite.config.js
+++ b/vite.config.js
@@ -16,8 +16,25 @@ export default defineConfig({
         }),
       ],
       output: {
-        manualChunks: {
-          vendor: ['dexie', 'react-dom/client', 'react-router'],
+        codeSplitting: {
+          groups: [
+            {
+              name: 'cards',
+              test: 'cards.json',
+            },
+            {
+              name: 'dexie',
+              test: 'dexie',
+            },
+            {
+              name: 'react-dom',
+              test: 'react-dom',
+            },
+            {
+              name: 'react-reconciler',
+              test: 'react-reconciler',
+            },
+          ],
         },
       },
     },

